app/tests/reader_studies_tests/test_api.py

```diff
@@ -1842,8 +1839,8 @@ def test_migrate_to_display_sets(client, settings):
     rs.add_editor(editor)
     rs.add_reader(reader)
     rs.view_content = {
-        "main": "generic-medical-image",
-        "secondary": "generic-overlay",
+        "main": ["generic-medical-image"],
+        "secondary": ["generic-overlay"],
```

Comment:
Shouldn't this be `{"main": ["generic-medial-image", "generic-overlay"]}`?

Reply:
That's handled in the block before this. This is to test multiple viewports and just using the interfaces that are available. I'd be happy to create a new interface for this case though.